src/FTGlyph/FTPixmapGlyph.cpp

```diff
@@ -40,18 +41,30 @@
 
 
 FTPixmapGlyph::FTPixmapGlyph(FT_GlyphSlot glyph) :
-    FTGlyph(new FTPixmapGlyphImpl(glyph))
+    FTGlyph(NewImpl(glyph))
 {}
 
 
 FTPixmapGlyph::~FTPixmapGlyph()
 {}
 
 
+FTGlyphImpl *FTPixmapGlyph::NewImpl(FT_GlyphSlot glyph)
+{
+  FTPixmapGlyphImpl *Impl = new FTPixmapGlyphImpl(glyph);
+  if (Impl->destWidth && Impl->destHeight)
```

Comment:
Can you please add a line of comment to explain what is the intent?

It seems you want to provide a fallback, in case ... what? (please explain in more higher-level terms than width or height is zero).